packages/dds/tree/src/simple-tree/schemaTypes.ts

```diff
@@ -476,9 +476,34 @@ function evaluateLazySchema(value: LazyItem<TreeNodeSchema>): TreeNodeSchema {
 }
 
 /**
- * Types allowed in a field.
+ * Types of {@link TreeNode|TreeNodes} or {@link TreeLeafValue|TreeLeafValue} allowed at a location in a tree.
```

Comment:
```suggestion
 * Types of {@link TreeNode|TreeNodes} or {@link TreeLeafValue|TreeLeafValues} allowed at a location in a tree.
```

Reply:
An alternative that might be simpler would be:
```suggestion
 * Types of {@link TreeNode}s or {@link TreeLeafValue}s allowed at a location in a tree.
```

But either way seems fine